ios/MaasCore/README.md

```diff
@@ -0,0 +1,3 @@
+# MaasCore
+
+A description of this package.
```

Comment:
😁 A brief description of the functionality this package (its products?) provides may be useful.